modules/nf-core/deepvariant/postprocessvariants/main.nf

```diff
@@ -1,14 +1,12 @@
-process DEEPVARIANT {
+process DEEPVARIANT_POSTPROCESSVARIANTS {
     tag "$meta.id"
-    label 'process_high'
+    label 'process_single'
```

Comment:
`nf-core/deepvariant:1.6.1` now exists, this could be `process_high`

Reply:
Fixed

Reply:
I have some second thoughts on this - I find that it spends a lot of time using a single CPU, even though it can indeed use multiple threads for some parts. I think it may be good to use process_medium as a compromise of allocation efficiency and reducing the runtime.